Sail/src/Sail/entities/Entity.h

```diff
@@ -9,19 +9,23 @@
 class Entity {
 public:
 	typedef std::shared_ptr<Entity> SPtr;
-	static SPtr Create();
+	static SPtr Create(const std::string& name = "");
 public:
 	Entity();
```

Comment:
```suggestion
```
Unnecessary constructor since the other one has a default value for the parameter

Reply:
Left it for future additions.